PolyEngine/Engine/Src/World.hpp

```diff
@@ -224,8 +230,9 @@ namespace Poly {
 	template<typename T>
 	T* Entity::GetComponent()
 	{
-		if (HasComponent(gEngine->GetComponentID<T>()))
-			return static_cast<T*>(Components[gEngine->GetComponentID<T>()]);
+		auto const ctypeID = ComponentsFamily::type_id<T>();
```

Comment:
change to `const auto`, do it in all occurences.

Reply:
done